Jenkinsfile

```diff
@@ -22,6 +22,7 @@ pipeline {
         stage('Init') {
             steps {
                 sh 'make clear-all || true'
+                sh 'docker image prune -a'
```

Comment:
Is there a reason not to do system prune?

Reply:
@YuviGold what do you think?

Reply:
I think this isn't a job duty to clean a machine. Especially for the day we would run parallel jobs simultaneously.

Reply:
Do we have some periodical job that can do it?

Reply:
no